neuralmonkey/readers/plain_text_reader.py

```diff
@@ -19,6 +23,38 @@ def reader(files: List[str]) -> Iterable[List[str]]:
     return reader
 
 
+def column_separated_reader(column: int, delimiter: str = "\t",
+                            quotechar: str = None, encoding: str = "utf-8"):
```

Comment:
to poznáš podle toho, co to vrací :-)

Tady v tom případě vracim funkci, která žere seznam stringů a vrací iterable přes listy stringů. Nechal bych ti to jako cvičení, ale nechci sekýrovat, takže tu:

```python3
def column_separated_reader(...) -> Callable[[List[str]],Iterable[List[str]]]
```

Možná tam mam blbě nějakou tu závorku. Callable má dva typový argumenty: seznam vstupních typů (tj. argumentů tý funkce) a návratovej typ.